### PR TITLE
Feat:minors algorithm for the MREstimator

### PR DESCRIPTION
**Pull request by Javierverbel**
### Description
- Make sure thst the minors algorithm is correctly added.

### CODE REVIEW

File: cryptographic_estimators/MREstimator/MRAlgorithms/minors.py [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+from ...MREstimator.mr_problem import MRProblem
+from ...base_algorithm import optimal_parameter
+from math import log2, ceil
+from math import comb as binomial
+from .mr_helper import minors_polynomial
+from ..mr_constants import MR_NUMBER_OF_KERNEL_VECTORS_TO_GUESS, MR_NUMBER_OF_COEFFICIENTS_TO_GUESS
+
+
+class Minors(MRAlgorithm):
+    r"""
```
Comment (FloydZ) [inline cryptographic_estimators/MREstimator/MRAlgorithms/minors.py:28]: remove the raw string

File: cryptographic_estimators/MREstimator/MRAlgorithms/minors.py [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+            sage: from cryptographic_estimators.MREstimator.MRAlgorithms.minors import Minors
+            sage: from cryptographic_estimators.MREstimator.mr_problem import MRProblem
+            sage: ME = Minors(MRProblem(q=16, m=15, n=15, k=78, r=6))
+            sage: ME.lv()
+            0
+        """
+        return self._get_optimal_parameter(MR_NUMBER_OF_COEFFICIENTS_TO_GUESS)
+
+    def _ME_time_complexity_helper_(self, m, n_reduced, k_reduced, r):
+        time = 0
```
Comment (FloydZ) [inline cryptographic_estimators/MREstimator/MRAlgorithms/minors.py:103]: the only difference between `_ME_time_complexity_helper` and `_ME_memory_complexity_helper` is `w*` and `2*`. So maybe we can merge those two into one function, to reduce code complexity.

**Review by FloydZ — COMMENTED** [review]
just small things

**Comment by Javierverbel** [comment]
@Javierverbel PR updated. Please approve if it is fine with you.

**Review by FloydZ — APPROVED** [review]

**Pull request merged**